### PR TITLE
fix: lf footer website guidelines

### PR DESCRIPTION
**Pull request by degenaro**
https://github.com/cncf/foundation/blob/main/website-guidelines.md

## Types of changes

- [ ] Hot fix (emergency fix and release)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Documentation (change which affects the documentation site)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Release (`develop` -> `main`)

## Quality assurance (all should be covered).

- [ ] My code follows the code style of this project.
- [ ] Documentation for my change is up to date?
- [ ] My PR meets testing requirements.
- [ ] All new and existing tests passed.
- [x] All commits are signed-off.

## Key links:

- [Sonar coverage](https://sonarcloud.io/dashboard?id=compliance-trestle)

# Before you merge

- Ensure it is a 'squash commit' if not a release.
- Ensure CI is currently passing
- Check sonar. If you are working for a fork a maintainer will reach out, if required.

### CODE REVIEW

File: README.md
Comment (jpower432) [inline README.md:112]: I think we should leave this as the `compliance-trestle` maintainers list

File: README.md
Comment (jpower432) [inline README.md:134]: Per the guidelines, this should also include the CNCF logo.

**Review by jpower432 — COMMENTED** [review]
Is the goal here to update the `compliance-trestle` website as well or just the README? The website is not changed by because there is a copy of the README here https://github.com/oscal-compass/compliance-trestle/blob/develop/docs/index.md

File: README.md
Comment (degenaro) [reply README.md:112]: Yes, of course! The lists need not be the same. I will fix here and the others umbrella projects.

File: README.md
Comment (degenaro) [reply README.md:139]: Yep. Fixed.

**Comment by degenaro** [comment]
> Is the goal here to update the `compliance-trestle` website as well or just the README? The website is not changed by because there is a copy of the README here https://github.com/oscal-compass/compliance-trestle/blob/develop/docs/index.md

Fair point. This should be updated as well. Kinda ugly to have essentially 2 copies.

**Comment by degenaro** [comment]
@jpower432 Thx for your careful review. My claim is that all these problems have been remedied.

File: README.md
Comment (butler54) [inline README.md:115]: What is the story on the headers in the python files? 

Do we need to replace those as well? @jpower432

File: docs/maintainers.md
Comment (butler54) [inline docs/maintainers.md:1]: This should not be changed. The mainters reference automatically reads the file at the root of the directory of the repository. That file should be changed instead.

**Review by butler54 — CHANGES_REQUESTED** [review]

**Comment by butler54** [comment]
> fair point. This should be updated as well. Kinda ugly to have essentially 2 copies.

I think in this context @degenaro we can de-duplicate by pulling in external markdown files. For the index file it's a little tricky as we might make relative link assumptions. I've updated a website epic #1677 with this.

**Comment by butler54** [comment]
Another question do we need to add this as a footer to all pages. It's a pretty easy config: https://squidfunk.github.io/mkdocs-material/setup/setting-up-the-footer/?h=footer#copyright-notice (maybe no the logo on every page).

File: README.md
Comment (degenaro) [reply README.md:115]: I've seen nothing the in the CNCF read and understand documentation for such a requirement. Just about acceptable licenses, which we have already.

File: docs/maintainers.md
Comment (degenaro) [reply docs/maintainers.md]: Fixed.

**Comment by degenaro** [comment]
> > fair point. This should be updated as well. Kinda ugly to have essentially 2 copies.
> 
> I think in this context @degenaro we can de-duplicate by pulling in external markdown files. For the index file it's a little tricky as we might make relative link assumptions. I've updated a website epic #1677 with this.

No need to fix now. Future fix in website epic is good idea.

**Comment by degenaro** [comment]
> Another question do we need to add this as a footer to all pages. It's a pretty easy config: https://squidfunk.github.io/mkdocs-material/setup/setting-up-the-footer/?h=footer#copyright-notice (maybe no the logo on every page).

The guidelines  https://github.com/cncf/foundation/blob/main/website-guidelines.md are not crystal clear. I think that if we had a footer on every page, then we'd need to update it. But using https://kyverno.io/ as a model they just have a footer on the homepage. I don't think we are required to add footers on every page.

**Review by yuji-watanabe-jp — APPROVED** [review]

File: README.md
Comment (jpower432) [reply README.md:115]: Here is some CNCF guidance I found around copyright [notices](https://github.com/cncf/foundation/blob/main/copyright-notices.md). I do think updating existing notices is something we should complete based this guidance and CNCF project convention, but this could be considered out of scope for this PR IMO.

Comment (degenaro) [reply README.md:115]: The guidance clearly states: `Please note that it is not wrong, and it is acceptable, if a contributor wishes to keep their own copyright notices on their contributions. The above is a recommended format for ease of use, but is not mandated by CNCF.`  My view is that we should leave the existing notices in place, and any new modules will get the new notice. However, it the majority feel uniformity is desirable I will not be a blocker.

Comment (jpower432) [reply README.md:115]: @degenaro I understand. I did not mean to imply this was a requirement. I read it as a recommendation. In any case, it seems we would need prior approval per this section of the guidance - `You should not change or remove someone else's copyright notice unless they have expressly permitted you to do so. This includes third parties' notices in pre-existing code.`

**Review by ancatri — APPROVED** [review]

**Review by jflowers — CHANGES_REQUESTED** [review]
Please remove Red Hat the footer

**Review by jflowers — APPROVED** [review]
abstain

**Review by butler54 — APPROVED** [review]
LGTM

**Pull request merged**
